packages/dbal-toggle/README.md

```diff
@@ -0,0 +1,20 @@
+# Pheature Flags Doctrine DBAL Toggle
+
+Pheature flags Doctrine DBAL toggle implementation
+
+## Installation
+
+Describe package installation
+
+```php
+composer require pheature/dbal-toggle
```

Comment:
Refer to file `composer.json` https://github.com/pheature-flags/pheature-flags/pull/98/files#r615274845